kivymd/uix/behaviors/elevation.py

```diff
@@ -149,32 +154,15 @@ def build(self):
             pos: self._hard_shadow_pos
         Color:
             a: 1
-
-
-<CircularElevationBehavior>
-    canvas.before:
-        Color:
-            a: self._soft_shadow_a
-        Rectangle:
-            texture: self._soft_shadow_texture
-            size: self._soft_shadow_size
-            pos: self._soft_shadow_pos
-        Color:
-            a: self._hard_shadow_a
-        Rectangle:
-            texture: self._hard_shadow_texture
-            size: self._hard_shadow_size
-            pos: self._hard_shadow_pos
-        Color:
-            a: 1
-"""
+""",
+    filename="CommonElevationBehavior.kv",
 )
 
 
 class CommonElevationBehavior(object):
     """Common base class for rectangular and circular elevation behavior."""
 
-    elevation = NumericProperty(1)
+    elevation = NumericProperty(None)
     """
     Elevation value.
```

Comment:
Elevation is set to None, this will allow us to know if the value is changed between python code instances and kvlang instances, if in neither of them the value is set, the elevation will be then set inside shadow_preset, this means that every subclass of the elevation behavior subclasses must set the default value by overwriting the instruction
```py
def shadow_preset(self,*dt):
    if self.elevation is None:
        self.elevation = 12 # new preset inside a subclass
    super().shadow_preset()
```